sdk/eventhub/eventhubs-checkpointstore-tables/package.json

```
@@ -0,0 +1,124 @@
+{
+  "name": "@azure/eventhubs-checkpointstore-tables",
+  "sdk-type": "client",
+  "version": "1.0.0-beta.1",
+  "description": "An Azure Storage Tables solution to store checkpoints when using Event Hubs.",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/eventhubs-checkpointstore-tables/",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "sideEffects": false,
+  "keywords": [
+    "azure",
+    "cloud",
+    "event hubs",
+    "events"
+  ],
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "main": "./dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "./typings/eventhubs-checkpointstore-tables.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
```

We recently changed the minimum version of node in our engines to `12.0.0`, so we should update this too.
```suggestion
    "node": ">=12.0.0"
```